create_todo/forms.py

```diff
@@ -0,0 +1,9 @@
+from django import forms
+
+
+class CreateForm(forms.Form):
+    """Form for creating a new todo"""
+    title = forms.CharField(max_length=150) 
+    description = forms.CharField(widget=forms.Textarea)
+    
+    
```

Comment:
`CreateForm` does not describe what it is creating. Also it will not only be used for creating. It will also be used for editing.